Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ruby:3.0
+
+ENV S3_KEY ""
+ENV S3_PRIVATE_KEY ""
+ENV FlAG "all"
+ENV EXT ""
+
+RUN mkdir -pv ~/.s3
+RUN apt update && apt install -y libmagic-dev
```

Comment:
I'm not sure taht libmagic-dev needed, it was required for working `ooxml_parser`